DOC-6104: Release notes third batch (#1843)
corda/corda-docs-portal: content/en/platform/corda/4.11/community/release-notes.md

An AES-key implementation is used to encrypt and decrypt distribution record recovery metadata for sharing and persistence amongst peers.

### DJVM removal

> With DJVM present, updates to Corda core must have also been compatible with

Reviewer: ```suggestion
The DJVM component required that all updates to Corda core were compatible with
```

> the `core-deterministic` module. The following changes mitigate this issue:

Reviewer: ```suggestion
the `core-deterministic` module. 
```

> * The experimental component DJVM has been removed from this and all future releases.

Reviewer: ```suggestion
To mitigate this issue, the experimental component DJVM has been removed from this and all future releases.
```

> * Because of the DJVM removal, the `DriverParameters` class has changed. The two constructor parameters `djvmBootstrapSource`

Reviewer: ```suggestion
* As a result of the DJVM removal, the `DriverParameters` class has changed. The two constructor parameters `djvmBootstrapSource`
```

> and `djvmCordaSource` have been removed from `DriverParameters`. Any client code using `DriverParameters` now needs at least recompiling.

Reviewer: ```suggestion
and `djvmCordaSource` have been removed from `DriverParameters`. Any client code using `DriverParameters` must be recompiled.
```

### DJVM removal

The DJVM component required that all updates to Corda core were compatible with
the `core-deterministic` module.
To mitigate this issue, the experimental component DJVM has been removed from this and all future releases.
As a result of the DJVM removal, the `DriverParameters` class has changed. The two constructor parameters `djvmBootstrapSource`

> and `djvmCordaSource` have been removed from `DriverParameters`. Any client code using `DriverParameters` now needs at least recompiling.

Author: Updated the syntax. Think it's clearer now.

> * Corda now supports JDK Azul 8u382 and Oracle JDK 8u381.

Reviewer: Doesn't seem like a fixed issue?

Author: Moved to new features and enhancements

> * log4j2.xml is now deleting the correct file for diagnostic and checkpoint logs in the rollover strategy configuration.

Author: Updated to present simple but not changing to not changing to a negative sentence at it might be confusing imo.

* Some log messages at the warning level, related to failed SSL handshakes, were inadvertently introduced during the improvements

>   to SSL certificate handling in the previous patch release. These messages would frequently appear in the logs as part of

Reviewer: ```suggestion
  to SSL certificate handling in the previous patch release. These messages frequently appeared in the logs as part of
```
